build/validate.sh

```diff
@@ -93,39 +94,24 @@ function checkDependencyViolationFinder {
 function checkWithLatestShop {
     echo "Checking module with latest $PRODUCT_NAME..."
 
-    composer config repositories.ecomodule path "$TRAVIS_BUILD_DIR/$MODULE_DIR"
```

Comment:
This code is used to check if CURRENT state of module is compatible with product.
Removing this will only validate module with updated dependencies.
I think we still need this code.